docker/unlock-integration.dockerfile

```diff
@@ -1,23 +1,25 @@
-FROM unlock
+FROM node:8.11.4
+MAINTAINER Unlock <ops@unlock-protocol.com>
```

Comment:
MAINTAINER is deprecated in favor of LABEL

Reply:
Thanks!